scripts/fetch-develop.deps.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Fetches the js-sdk and matrix-react-sdk dependencies for development
+# or testing purposes
+# If there exists a branch of that dependency with the same name as
+# the branch the current checkout is on, use that branch. Otherwise,
+# use develop.
+
+# Look in the many different CI env vars for which branch we're
+# building
+if [[ "$TRAVIS" == true ]]; then
+    curbranch="${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}"
```

Comment:
this, I suspect, isn't going to work when the PR is from a remote repo. I guess that's ok though.